pkg/proxy/ipvs/proxier.go

```diff
@@ -171,36 +170,27 @@ type IPGetter interface {
 	NodeIPs() ([]net.IP, error)
 }
 
+// realIPGetter is a real NodeIP handler, it implements IPGetter.
 type realIPGetter struct{}
 
 func (r *realIPGetter) NodeIPs() (ips []net.IP, err error) {
```

Comment:
This function could use a comment explaining what it does, and what the heuristic is.

Reply:
It also seems to have no tests

Reply:
> This function could use a comment explaining what it does, and what the heuristic is.

Sounds fair, added now. PTAL.

Reply:
> It also seems to have no tests

It's fixed now. PTAL.